spec/system/page_objects/components/topic_list_controls.rb

```diff
@@ -11,7 +11,7 @@ def has_new?(count:)
             I18n.t("js.filters.new.title_with_count", count: count)
           end
 
-        has_css?(".nav-item_new", text: text)
+        has_css?(".nav-item_new", exact_text: text)
```

Comment:
This is tricky because if you don't use exact text, `New` will match `New (1)` which is why we failed to catch the regression.